api-ref-assets/config/fields.yaml

```diff
@@ -593,7 +596,7 @@
     - volumeName
     - storageClassName
     - volumeMode
-  - name: Beta level
+  - name: Alpha level
```

Comment:
This is an odd change.

Reply:
I agree, but this is reflecting the change of the comment of the field

Reply:
I have pushed a commit to change Alpha -> Beta in the fields.yaml file.

Note that the generated HTML content is OK, as it was already containing Beta.